addons/web/static/tests/core/dropdown_tests.js

```diff
@@ -919,4 +920,19 @@ QUnit.module("Components", ({ beforeEach }) => {
         assert.containsNone(parent.el, ".dropdown-menu", "all menus are now closed");
         assert.strictEqual(hotkeyRegistrationsCount, 0, "no hotkey registration left");
     });
+
+    QUnit.test("Dropdown with a tooltip", async (assert) => {
+        assert.expect(2);
+
+        class MyComponent extends owl.Component {}
+        MyComponent.template = owl.tags.xml`
+            <Dropdown tooltip="'My tooltip'">
+                <DropdownItem/>
+            </Dropdown>`;
+
+        const parent = await makeParent(MyComponent);
```

Comment:
I don't think you should use this helper.
This test should only assert that the [data-tooltip] attribute is there in the dom.

This is the concern of tooltip tests to check if an element with a [data-tooltip] attribute renders a tooltip on mouseenter.

That way, you won't have to export the local tooltip helper function "makeParent".